bots/discord/jmusicbot/egg-j-music-bot.json

```diff
@@ -1,58 +1,130 @@
             "container": "debian:buster-slim",
-            "entrypoint": "\/bin\/bash"
+            "entrypoint": "bash"
```

Comment:
This should stay `/bin/bash` as poisoning the env vars is still potentially possible.

Reply:
Good catch. I was just trying to simplify and match other eggs I've worked on. I'll fix it soon.